inst/include/Rcpp/platform/compiler.h

```diff
@@ -56,6 +43,13 @@
     #if GCC_VERSION >= 40600
         #define IS_GCC_460_OR_LATER
     #endif
+    // Inform users of dated compiler (e.g. <= RHEL6 )
```

Comment:
Right, your hand was forced there.

I should note that I am not at all happy about the state of affairs with
RHEL 6 and we are looking at creating a new binary distribution of R for
RHEL 6 that uses clang. Until then though I don't think we can cry wolf
when being compiled on RHEL 6, it's just too mainstream a platform.

On Tue, Aug 2, 2016 at 4:50 PM, Dirk Eddelbuettel <notifications@github.com>
wrote:

> In inst/include/Rcpp/platform/compiler.h
> <https://github.com/RcppCore/Rcpp/pull/515#discussion_r73234667>:
>
> > @@ -56,6 +43,13 @@
> >      #if GCC_VERSION >= 40600
> >          #define IS_GCC_460_OR_LATER
> >      #endif
> > +    // Inform users of dated compiler (e.g. <= RHEL6 )
>
> (I should add that RcppArmadillo now actually fails when this happens, but
> then again Conrad left me no choice. It has 246 rev.deps and mayhem may
> well ensue ...)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RcppCore/Rcpp/pull/515/files/4bc57824b83491162a38e18fc3ca68d5e5ec3c41#r73234667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXx3FxbIiLaRmRSUxXk1fKSR09SYkFks5qb62agaJpZM4JTRmr>
> .
>